lwallet/Pos/Wallet/Light/Mode.hs

```diff
@@ -135,4 +136,8 @@ instance HasCoreConstants => MonadTxHistory LightWalletSscType LightWalletMode w
 
 instance MonadAddresses LightWalletMode where
     type AddrData LightWalletMode = PublicKey
-    getNewAddress pk = pure (makePubKeyAddress pk, Just $ addressHash pk)
+    getNewAddress pk = do
+        -- Light wallet doesn't know current slot, so let's assume
+        -- it's 0-th epoch. It's enough for our current needs.
+        ibea <- IsBootstrapEraAddr <$> gsIsBootstrapEra 0
```

Comment:
Currently this switch is achieved by setting unlock epoch to 0 in benchmarks mode and it works fine.